de-lib/src/main/java/org/iplantc/de/commons/client/validators/DiskResourceNameValidator.java

```diff
@@ -63,7 +61,7 @@ public DiskResourceNameValidator() {
         }
 
         if (restrictedFound.length() > 0) {
-            String errorMsg = validationMessages.drNameValidationMsg(new String(restrictedChars)) + " " //$NON-NLS-1$
+            String errorMsg = validationMessages.drNameValidationMsg(validationConstants.restrictedDiskResourceNameChars() + validationConstants.newlineToPrint() + validationConstants.tabToPrint()) + " " //$NON-NLS-1$
```

Comment:
In `CmdLineArgCharacterValidator`, you did a check first to see if newline and tab were in the restricted list.  I think it might be a good idea to have that in all the other cases as well.